rpc/cloudagent/compute/moc_cloudagent_placementgroup.pb.go

```diff
@@ -0,0 +1,477 @@
+	xxx_messageInfo_PlacementGroupPrecheckRequest.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_PlacementGroupPrecheckRequest proto.InternalMessageInfo
+
+func (m *PlacementGroupPrecheckRequest) GetPlacementGroups() []*PlacementGroup {
+	if m != nil {
+		return m.PlacementGroups
+	}
+	return nil
+}
+
+type PlacementGroupPrecheckResponse struct {
+	// The precheck result: true if the precheck criteria is passed; otherwise, false
+	Result *wrappers.BoolValue `protobuf:"bytes,1,opt,name=Result,proto3" json:"Result,omitempty"`
+	// The error message if the precheck is not passed; otherwise, empty string
+	Error                string   `protobuf:"bytes,2,opt,name=Error,proto3" json:"Error,omitempty"`
+	XXX_NoUnkeyedLiteral struct{} `json:"-"`
+	XXX_unrecognized     []byte   `json:"-"`
+	XXX_sizecache        int32    `json:"-"`
+}
+
+func (m *PlacementGroupPrecheckResponse) Reset()         { *m = PlacementGroupPrecheckResponse{} }
+func (m *PlacementGroupPrecheckResponse) String() string { return proto.CompactTextString(m) }
+func (*PlacementGroupPrecheckResponse) ProtoMessage()    {}
+func (*PlacementGroupPrecheckResponse) Descriptor() ([]byte, []int) {
+	return fileDescriptor_0f298d020ddbfe82, []int{3}
+}
+
+func (m *PlacementGroupPrecheckResponse) XXX_Unmarshal(b []byte) error {
+	return xxx_messageInfo_PlacementGroupPrecheckResponse.Unmarshal(m, b)
+}
+func (m *PlacementGroupPrecheckResponse) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
+	return xxx_messageInfo_PlacementGroupPrecheckResponse.Marshal(b, m, deterministic)
+}
+func (m *PlacementGroupPrecheckResponse) XXX_Merge(src proto.Message) {
+	xxx_messageInfo_PlacementGroupPrecheckResponse.Merge(m, src)
+}
+func (m *PlacementGroupPrecheckResponse) XXX_Size() int {
+	return xxx_messageInfo_PlacementGroupPrecheckResponse.Size(m)
+}
+func (m *PlacementGroupPrecheckResponse) XXX_DiscardUnknown() {
+	xxx_messageInfo_PlacementGroupPrecheckResponse.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_PlacementGroupPrecheckResponse proto.InternalMessageInfo
+
+func (m *PlacementGroupPrecheckResponse) GetResult() *wrappers.BoolValue {
+	if m != nil {
+		return m.Result
+	}
+	return nil
+}
+
+func (m *PlacementGroupPrecheckResponse) GetError() string {
+	if m != nil {
+		return m.Error
+	}
+	return ""
+}
+
+type PlacementGroup struct {
+	Name                 string         `protobuf:"bytes,1,opt,name=name,proto3" json:"name,omitempty"`
+	Id                   string         `protobuf:"bytes,2,opt,name=id,proto3" json:"id,omitempty"`
+	GroupName            string         `protobuf:"bytes,3,opt,name=groupName,proto3" json:"groupName,omitempty"`
+	Type                 string         `protobuf:"bytes,4,opt,name=type,proto3" json:"type,omitempty"`
+	Scope                string         `protobuf:"bytes,5,opt,name=scope,proto3" json:"scope,omitempty"`
+	Zones                []string       `protobuf:"bytes,6,rep,name=zones,proto3" json:"zones,omitempty"`
```

Comment:
use the 'ZoneConfiguration' as defined in VM in placement group PR. Hopefully that will get checked-in first, till then you can temporarily add it here.

Reply:
Added ZoneConfiguration instead of zones